bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/api/dto/HomeStatusModule.java

```diff
@@ -95,8 +96,8 @@ public SirenStatus getSirenStatus() {
         return sirenStatus;
     }
 
-    public @Nullable String getVpnUrl() {
-        return vpnUrl;
+    public String getVpnUrl() {
+        return Objects.requireNonNull(vpnUrl);
```

Comment:
Why `vpnUrl` would be always not null ?

Reply:
Yes, maybe is it too much. From class perspective vpnUrl can be null as this class is generic to various modules. But when calling this method it is expected not to be null. This can be misleading, I'll revert.